source/guides/clear/autoproxy.rst

```diff
@@ -101,42 +107,48 @@ Unmask the :command:`pacrunner` service by running:
 
    systemctl unmask pacrunner.service
 
-:command:`FindProxyForURL` with :command:`busctl` can also indicate if the
-:command:`pacrunner.service` is masked.
 
-.. code-block:: bash
 
-   busctl call org.pacrunner /org/pacrunner/client org.pacrunner.Client 
+Use :command:`FindProxyForURL` with :command:`busctl` to indicate |br|
 
-.. code-block:: console
-   
-   FindProxyForURL ss "http://www.google.com" "google.com"
-   Unit pacrunner.service is masked.
-   dig wpad, dig wpad.<domain>
+#. if the :command:`pacrunner.service` is masked:
 
-:command:`FindProxyForURL` returns the URL and port of the proxy server when
-an external URL and host are provided as arguments.
+   .. code-block:: bash
 
-.. code-block:: bash
+      busctl call org.pacrunner /org/pacrunner/client org.pacrunner.Client FindProxyForURL ss "http://www.google.com" "google.com"
 
-   busctl call org.pacrunner /org/pacrunner/client org.pacrunner.Client 
+   Sample output:
 
-.. code-block:: console
+   .. code-block:: console
 
-   FindProxyForURL ss "http://www.google.com" "google.com"
-   s "PROXY proxy.your.domain.com:<port>"
+      Unit pacrunner.service is masked.
+      dig wpad, dig wpad.<domain>
 
-If a proxy server is not avialable, or if :command:`pacrunner` is running
-without a PAC file, :command:`FindProxyForURL` will return "DIRECT". 
+#. the URL and port of the proxy server when an external URL and host are
+   provided as arguments:
 
-.. code-block:: bash
+   .. code-block:: bash
+
+      busctl call org.pacrunner /org/pacrunner/client org.pacrunner.Client FindProxyForURL ss "http://www.google.com" "google.com"
 
-   busctl call org.pacrunner /org/pacrunner/client org.pacrunner.Client 
+   Sample output:

```

Comment:
Could you revise this slightly to indicate that in this case, `Sample output` indicates **success**? 
I think it will help the reader understand the goal of the doc better.

Reply:
Thanks @mvincerx  - I made some edits to improve clarity.